README.md

```diff
@@ -2,16 +2,40 @@
 
 [![Build Status](https://travis-ci.org/hmrc/api-subscription-fields.svg)](https://travis-ci.org/hmrc/api-subscription-fields) [ ![Download](https://api.bintray.com/packages/hmrc/releases/api-subscription-fields/images/download.svg) ](https://bintray.com/hmrc/releases/api-subscription-fields/_latestVersion)
 
-This microservice stores definitions and values for the HMRC Developer Hub.
+This microservice stores and retrieves subscription fields and their values.
```

Comment:
If you write the word "subscription", you should explain what it is.
But would be too complex to explain in few words.
We prefer to do not mention that word.
But we want to mention the HMRC Developer Hub.
I suggest to just leave this:
```
This microservice stores definitions and values for the HMRC Developer Hub.
```
If you want to mention the word "subscription", you need to write more...